src/models/appointmentService.js

```diff
@@ -10,7 +10,14 @@ Bahmni.Appointments.AppointmentService = (function () {
         var dateUtil = Bahmni.Common.Util.DateUtil;
 
         var getTime = function (dateTime) {
-            return dateTime ? dateUtil.getDateTimeInSpecifiedFormat(dateTime, timeFormat) : undefined;
+            if (!dateTime) {
+                return undefined;
+            }
```

Comment:
Return the time in UTC formatted as 'HH:mm:ss'.
Current date is used together with the passed time before converting UTC so that the returned time is ok whether we're currently on daylight saving time or not.